Finished grooming game loop

johnzillakaiju opened the pull request:
still need to add sprite changes at certain HP threshold and polish

Commit:
Finished grooming game loop

still need to add sprite changes at certain HP threshold and polish

VeoWulfOfWallSt reviewed (dismissed):
Able to use the tools when the curtain is up

khmflrdlz reviewed (approved):
No issue with tool changes. the game loop is working properly.
Pull Request approved.

VeoWulfOfWallSt committed:
Added dog Stats to Grooming

Stats are now functional in Grooming

Commit:
Sprite can now change sprite on 50 hp in every tool

Commit:
Fixed Scaling problem in Grooming Mini Game

deleted some unused game object so prepare for conflicts in merging

Commit:
Applied changes to DogData class and Made changes to GetDogSo

Added a int Beauty for dogdata, added more functionality to GetDogSo

Commit:
Fixed some issue on GetSo Multiplying stats every load

also added a string filename in DogSo

Commit:
Added selection manager in main menu

Commit:
Data persistent is done

added a singleton for selector manager in play park and added a script that can load the reference SO in the singleton I've already created this gameobject in agility mini game please see

VeoWulfOfWallSt committed:
Added Grooming data load and save

VeoWulfOfWallSt reviewed (approved):
General Gameplay loop is working, Approving for now

Known Issue: The user is still able to use the tool during tool transitions
